tests/system/HTTP/IncomingRequestTest.php

```diff
@@ -709,66 +712,75 @@ public function testGetIPAddressNormal()
     public function testGetIPAddressThruProxy()
     {
         $expected                        = '123.123.123.123';
+        $_SERVER['HTTP_X_FORWARDED_FOR'] = $expected;
         $_SERVER['REMOTE_ADDR']          = '10.0.1.200';
         $config                          = new App();
         $config->proxyIPs                = '10.0.1.200,192.168.5.0/24';
-        $_SERVER['HTTP_X_FORWARDED_FOR'] = $expected;
-        $this->request                   = new Request($config);
+
+        $this->request = new Request($config);
+        $this->request->populateHeaders();
 
         // we should see the original forwarded address
         $this->assertSame($expected, $this->request->getIPAddress());
     }
 
     public function testGetIPAddressThruProxyInvalid()
     {
-        $expected                        = '123.456.23.123';
-        $_SERVER['REMOTE_ADDR']          = '10.0.1.200';
+        $_SERVER['HTTP_X_FORWARDED_FOR'] = '123.456.23.123';
+        $expected                        = '10.0.1.200';
+        $_SERVER['REMOTE_ADDR']          = $expected;
```

Comment:
Double assign?
```suggestion
        $_SERVER['HTTP_X_FORWARDED_FOR'] = '123.456.23.123';
        $_SERVER['REMOTE_ADDR']          = $expected = '10.0.1.200';
```

Reply:
This code seems a bit odd to me, because of the spaces to the left of the first `=`.

Reply:
Very not necessary.